src/maketype.jl

```diff
@@ -414,11 +407,38 @@ function addjumps!(rn::DiffEqBase.AbstractReactionNetwork;
                                                           syms_to_ints,
                                                           params_to_ints;
                                                           minimal_jumps=minimal_jumps)
-    
+
     rn.jump_rate_expr   = jump_rate_expr
     rn.jump_affect_expr = jump_affect_expr
     rn.jumps            = build_jumps ? eval(jumps) : nothing
     rn.regular_jumps    = build_regular_jumps ? eval(get_regularjumps(reactions, syms_to_ints, params_to_ints)) : nothing
 
     nothing
 end
+
+#Adds stuff required to do homotopy continuation based equilibration calculations.
+function addequi1!(rn::DiffEqBase.AbstractReactionNetwork)
+    @unpack params, reactions, syms_to_ints, params_to_ints, scale_noise = rn
+
+    # first construct an ODE reaction network (required for getting f_func).
+    if rn.f == nothing
+        addodes!(rn)
```

Comment:
I wonder if simply requiring the user to call `addodes!` before calling `addequil` would allow merging `addequil1` and `addequil2`? It seems reasonable to require this seeing as one is looking properties of the SS ODE solution.

Reply:
Nevermind, I missed that @TorkelE tried this and it didn't work...